src/components/Dropdown.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  debounceDelay: number;
+  onSelected: (person: Person) => void;
+};
+
+function debounce(
+  callback: React.Dispatch<React.SetStateAction<string>>,
+  delay: number,
+) {
+  let timerId = 0;
+
+  return (...args: any) => {
```

Comment:
Try to use `unknown` type

Reply:
Here I get errors 
<img width="666" alt="Screenshot 2023-07-31 at 15 45 29" src="https://github.com/mate-academy/react_autocomplete/assets/107880739/71c076b0-6551-4ded-82c8-c51370d14c0c">
<img width="729" alt="Screenshot 2023-07-31 at 15 45 49" src="https://github.com/mate-academy/react_autocomplete/assets/107880739/efcb26cc-4ac9-4e9a-a4a1-c6b9207ab571">